api/api.go

```diff
@@ -130,7 +136,50 @@ func (w Wrapper) GetIRMAAuthenticationResult(ctx echo.Context, sessionToken stri
 	base64String := base64.StdEncoding.EncodeToString(bytes)
 	sessionID := w.APIAuth.StoreVP(customerID, base64String)
 
-	newToken, err := w.APIAuth.CreateSessionJWT(customerID, sessionID)
+	newToken, err := w.APIAuth.CreateSessionJWT(customerID, sessionID, true)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError, err)
+	}
+
+	return ctx.JSON(200, domain.SessionToken{Token: string(newToken)})
+}
+
+func (w Wrapper) AuthenticateWithDummy(ctx echo.Context) error {
```

Comment:
Can't we split this up into multiple authentication backends? With an interface

Reply:
Sure, but I'm not certain adding an interface, and two implementations really makes things that much cleaner.

Reply:
For a non-demo application I would say it makes sense as for the API it shouldn't really matter how authentication takes place which makes it a nice candidate for an abstraction.